static_src/stores/user_store.js

```diff
@@ -164,6 +164,13 @@ export class UserStore extends BaseStore {
         break;
       }
 
+      case userActionTypes.USER_REMOVED_ALL_SPACE_ROLES: {
```

Comment:
only need one more test. that it changes the list of users upon removing the person.